python/examples/001-init-notcurses.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# Copyright 2020 igo95862
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from time import sleep
+
+from notcurses import get_std_plane
+
+std_plane = get_std_plane()
+
+sleep(1)
```

Comment:
Why the sleep here? Were you seeing failures otherwise? I'd rather not introduce a second of latency to every application that uses us. That would likely cause me to reach for a different library.

Reply:
ohhhhhhhhhhhhhh, this is example code. OK, what I would do, then, is use the `NOTCURSES_NO_ALTERNATE_SCREEN` ncoption. I assume you have this `sleep()` to make the output actually visible? Yeah, use the `NO_ALTERNATE_SCREEN`, please.

Reply:
Basically after you render something you need sleep for it to stay on screen. Otherwise once you reach the end of the script the context and all planes will get deferenced and the screen will revert to original state. It should be possible to include more advanced event loop in the future. The input test example has a state tied to inputs so it does not use sleep.